secret-service/pkg/backend/scopebackend_k8s.go

```diff
@@ -0,0 +1,19 @@
+package backend
+
+import "sort"
+
+func (k K8sSecretBackend) GetScopes() ([]string, error) {
+	scopes, err := k.ScopesRepository.Read()
+	if err != nil {
+		return nil, err
+	}
+	scopeArray := make([]string, len(scopes.Scopes))
+
+	i := 0
+	for scope := range scopes.Scopes {
+		scopeArray[i] = scope
+		i++
+	}
+	sort.Strings(scopeArray)
```

Comment:
This can be simplified... will comment with a better solution in a second

Reply:
```suggestion
	for scope := range scopes.Scopes {
	        scopeArray = append(scopeArray, scope)
	}
	sort.Strings(scopeArray)
```

Reply:
If the slice is pre-created we can't use this syntax.
On create `scopeArray = ["", "", ""]`
With append, the result will be `scopeArray = ["", "", "", "keptn-default", "keptn-webhook service", "dynatrace-service"]`